src/core/theme/utils/outline.ts

```diff
@@ -50,7 +46,7 @@ const afterBoxshadow = ({
 
 // TODO Refactor, create interfaces (and extend with Partial<>), add JSDOC for functions
 export const boxshadowOutline = ({
-  color = colors.accentBase,
+  color = colors.accentSecondary,
   borderColor = 'white',
```

Comment:
Not changed in this PR, but would it be possible to get (`white`) it from theme/tokens?

Reply:
That would be better, I forgot to change that after initial tests.